manual/statements/prepared/README.md

```diff
@@ -241,9 +241,27 @@ Changing the driver's defaults should be done with care and only in
 specific situations; read each method's Javadoc for detailed
 explanations.
 
+### Avoid preparing 'select *' queries
+
+Both the driver and Cassandra maintain a mapping of `PreparedStatement` queries to their
+metadata.  When a change is made to a table, such as a column being added or dropped, there
+is currently no mechanism for Cassandra to invalidate the existing metadata.  Because of this,
+the driver is not able to properly react to these changes and will improperly read rows after
+a schema change is made.
+
+Therefore it is currently recommended to not create prepared statements
+for 'select *' queries unless you don't plan on making schema changes involving
+adding or dropping columns. Alternatively you should list all columns of interest
```

Comment:
I would replace "unless you don't plan" with "if you plan" (the double negative is harder to read).